cli/cmd/env/configure.go

```diff
@@ -95,6 +78,7 @@ func ConfigureCmd(ch *cmdutil.Helper) *cobra.Command {
 	configureCommand.Flags().SortFlags = false
 	configureCommand.Flags().StringVar(&projectPath, "path", ".", "Project directory")
 	configureCommand.Flags().StringVar(&projectName, "project", "", "")
+	configureCommand.Flags().StringVar(&environment, "environment", "", "Environment for which variables apply (options: 'production', 'development', '')")
```

Comment:
The options are `dev` and `prod`:

```suggestion
	configureCommand.Flags().StringVar(&environment, "environment", "", "Optional environment to resolve for (options: dev, prod)")
```